### PR TITLE
docs: Improve table in install README

### LINKED ISSUE #276: Improve table in install README
Improve the table in the doc below to make it clearer which option is appropriate for each use-case:

- https://github.com/kata-containers/documentation/blob/master/install/README.md

### PR DESCRIPTION
**Pull request by jodh-intel**
Improvements to the table in the installation README:

- Fix the invalid link for the "Build from sources" option.
- Add column for "Packaged install" to make it clearer which
options result in a distro-packed install.
- Tweaked the "Suggested for" column to make the use-case options
clearer.
- Added detail for each use-case in a "Description" column.

Also, added a little more information on the "Scripted Installation" section of the install README.

Fixes #276.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Ummm

```
INFO: Checking all document URLs
ERROR: Invalid URL 'https://github.com/kata-containers/tests/data/rpm-signkey.pub' found in the following files:
install/README.md
```

**Comment by jodh-intel** [comment]
@marcov - ^ ;)

**Comment by jodh-intel** [comment]
Thanks - should it be in the packaging repo though I wonder?

**Comment by marcov** [comment]
Not sure, I thought to put it here since this repo is already pulled for automating install with kata-manager & co

**Comment by jodh-intel** [comment]
Fair point :)

/cc @jcvenegas.

**Comment by marcov** [comment]
@jodh-intel kata-containers/tests#825

**Comment by jodh-intel** [comment]
Blocked on https://github.com/kata-containers/tests/pull/825.

**Comment by marcov** [comment]
@jodh-intel please also fix the key url to https://raw.githubusercontent.com/kata-containers/tests/master/data/rpm-signkey.pub

**Comment by jodh-intel** [comment]
@marcov - done.

**Comment by jodh-intel** [comment]
Ping @kata-containers/documentation

File: install/README.md [hunk 1]
```diff
@@ -19,18 +19,28 @@ to see if your system is capable of running Kata Containers.
...
+| [Manual](#manual-installation)                             | Self paced user install     | Allows the user to read a brief document and exectute the specified commands step-by-step.                                                  | yes              | [see table](#supported-distributions) |
+| [Scripted](#scripted-installation)                         | Administrators              | Generates an installation script which will result in a working system when executed.                                                       | yes              | [see table](#supported-distributions) |
+| [Build from sources](../Developer-Guide.md#initial-setup) | Developers and hackers only | Allows power users who are comfortable building software from source to use the latest component versions. Not recommended for normal users. | no               | any distro                            |
+
+> **Notes:**
+>
+> - The "Packaged install" column shows if the resulting installation
+>   uses your distributions native package format (such as RPM or DEB).
```
Comment (intelkevinputnam) [inline install/README.md]: Line 33: ... distribution's

File: install/README.md [hunk 2]
```diff
@@ -19,18 +19,28 @@ to see if your system is capable of running Kata Containers.
...
+> - Power users who decide to build from sources should be aware of the
+>   implications of using an unpackaged system which will not be automatically
+>   updated as new [releases](../Releases.md) are made available.
 
 ### Automatic Installation
 [Use kata-manager](installing-with-kata-manager.md) to automatically install Kata packages.
 
 ### Scripted Installation
-[Use kata-doc-to-script](installing-with-kata-doc-to-script.md) to generate installation scripts.
+[Use kata-doc-to-script](installing-with-kata-doc-to-script.md) to generate installation scripts which can be reviewed before they are executed.
```
Comment (intelkevinputnam) [inline install/README.md]: Line 43: ... that can be reviewed ...

**Comment by jodh-intel** [comment]
Thanks @intelkevinputnam - branch updated.

**Comment by jodh-intel** [comment]
Ping @kata-containers/documentation - this needs to land today.

**Comment by jodh-intel** [comment]
Oops - @intelkevinputnam - I merged this before you'd ack'd. Let me know if you're not happy and I can raise another PR but I have applied all your feedback.

**Pull request merged**